Source/ReadEosTable.cpp

```diff
@@ -7,8 +7,12 @@
 
 #include "EosTable.H"
 
+#ifdef AMREX_USE_MPI
 // mini NoMPI
-#define HAVE_CAPABILITY_MPI //FIXME: This should be defined only when USE_MPI = TRUE
+#define HAVE_CAPABILITY_MPI 
```

Comment:
Why not just use AMREX_USE_MPI instead of creating another precompiler definition?

Reply:
Done!